docs/man/policy/attributes/list.md

```diff
@@ -2,6 +2,9 @@
 title: List attributes
 command:
   name: list
+  aliases:
+    - ls
+    - l
```

Comment:
Looks like we have ls and l both for LIST of namespaces, attributes, and values, but only l for kas-registry, subject-mappings, subject-condition-sets, and resource-mappings. Is that desired behavior?